easy_entrez/api.py

```diff
@@ -67,27 +67,39 @@ def is_response_for(response: EntrezResponse, query: Type[EntrezQueryT]) -> Type
 class EntrezAPI:
     """
     Parameters:
-        tool: Name of application making the E-utility call. Value must be a string with no internal spaces.
-        email: E-mail address of the E-utility user.
+        tool: Name of application making the E-utility call, for NCBI internal tracking.
+            Value must be a string with no internal spaces.
+            Example values are "easy-entrez" or "biopython".
```

Comment:
I would not want to recommend any specific values; downstream users can build a custom "tool" using this package as a dependency and they should specify their own tool name.

```suggestion
```

Reply:
Thanks for the comment! From my side, a large part of the reason I made this PR was because I felt `tool` was the most vague arg for `EntrezAPI`. As a new Entrez user, I had no idea what a "tool", now I understand it's just metadata NCBI uses. The value of `tool` can be any string really.

Biopython users get defaulted to using `"biopython"` as their tool: https://github.com/biopython/biopython/blob/biopython-181/Bio/Entrez/__init__.py#L141

I think it's important this docstring provide some concrete examples, I will adjust the wording to make it clear they're examples and not recommendations.